internal/handlers/pg/pgdb/pool_test.go

```diff
@@ -198,41 +198,54 @@ func TestConcurrentCreate(t *testing.T) {
 	schemaName := testutil.SchemaName(t)
 	tableName := testutil.TableName(t)
 
-	for _, withTable := range []bool{false, true} {
-		start := make(chan struct{})
-		res := make(chan error, n)
-		for i := 0; i < n; i++ {
-			go func() {
-				<-start
-				if withTable {
-					res <- pool.CreateTable(ctx, schemaName, tableName)
-				} else {
-					res <- pool.CreateSchema(ctx, schemaName)
-				}
-			}()
-		}
+	for _, tc := range []struct {
```

Comment:
@rumyantseva helped me refactor this test and make it work. Thanks a lot!
@AlekSi PTAL, maybe you have some thoughts on it.

Reply:
Well, at least it worked on my machine..

Reply:
This test was a bit strange initially... I wonder if we need to rewrite it.